fastlane/Fastfile

```diff
@@ -178,7 +178,7 @@ platform :ios do
   end
 
   desc 'Download and configure code signing certificates and provisioning profiles for the App Store build.'
-  lane :configure_code_signing_app_store do |options|
+  lane :configure_code_signing_app_store do |readonly: true|
```

Comment:
The first thing that comes to mind is that in other repos, we also replaced `desc` with YARD docs.
This PR already looks good to me as the main change are the keyword parameters, but I wonder if we should also standardize the comments.

Reply:
That's a good point. I'll take not to clean that up in a future PR (I already planned to make another pass at the YARD doc of all the lanes at some point anyway) 👍